integration-tests/Makefile

```diff
@@ -192,6 +192,12 @@ test_reorg_automation: ## Run the automation reorg tests
 build_docker_image:
 	docker build -f ../core/chainlink.Dockerfile --build-arg COMMIT_SHA=$(git rev-parse HEAD) --build-arg CHAINLINK_USER=chainlink -t $(image):$(tag) ../
 
+# image: the name for the chainlink image being built, example: image=chainlink
+# tag: the tag for the chainlink image being built, example: tag=latest
+# example usage: make build_docker_image image=chainlink tag=latest
+.PHONY: build_plugin_docker_image
+build_docker_image:
```

Comment:
This is the same target as the one above, I think it's supposed to be build_plugin_docker_image

Reply:
Yep, TY for pointing out